rental.rb

```diff
@@ -0,0 +1,19 @@
+require 'time'
+
+class Rental
+  attr_accessor :person, :book, :date
+
+  def initialize(person, book)
+    @date = DateTime.now.strftime('%Y/%m/%d %H:%M')
+    @person = person
+    @book = book
+  end
+
+  def assign_book
+    @book.rentals << self unless @books.rentals.include? self
+  end
+
+  def assign_person
+    @person.rentals << self unless @person.rentals.include? self
+  end
```

Comment:
- [ ] As requested by the previous reviewer, kindly implement the belongs-to relationship between Book/Rental and Person/Rental in the constructor 👍